jbrowse/src/org/labkey/jbrowse/JBrowseLuceneSearch.java

```diff
@@ -65,6 +70,10 @@ public class JBrowseLuceneSearch
     private final String[] specialStartPatterns = {"*:* -", "+", "-"};
     private static final String ALL_DOCS = "all";
     private static final String GENOMIC_POSITION = "genomicPosition";
+    private static final int maxCachedQueries = 25;
```

Comment:
If anything, I think i would make maxCachedQueries a much larger number, maybe 1000? I would guess that we want to constrain this by RAM (which is important), but I dont think we care about the absolute number of queries we cache